app/models/event.rb

```diff
@@ -2,6 +2,8 @@ class Event < ApplicationRecord
   EVENT_TYPES = %w[
     admin_creates_induction_period
     admin_updates_induction_period
+    admin_fails_teacher
+    admin_passes_teacher
```

Comment:
I think these  should contain the word `induction` because so there's no confusion on what is being passed or failed.